README.md

```diff
@@ -653,9 +653,9 @@ IEEE Conf. Computer Vision and Pattern Recognition (CVPR), 2020.
 - <a name="Li19neucom"></a>Li, H., Li, G., Shi, L.,  
 *[Super-resolution of spatiotemporal event-stream image](https://doi.org/10.1016/j.neucom.2018.12.048)*,  
 Neurocomputing, vol. 335, pp. 206-214, 2019. [PDF pre-print](https://arxiv.org/abs/1802.02398)
-- <a name="Mostafavi19arxiv"></a>Mostafavi I., S.M.,  Choi, J., Yoon, K.J,  
+- <a name="Mostafavi19arxiv"></a>Mostafavi I., S.M.,  Choi, J., Yoon, K.-J.,  
 *[Learning to Super Resolve Intensity Images from Events](https://arxiv.org/abs/1912.01196)*,  
-arXiv, 2019.
+IEEE Conf. Computer Vision and Pattern Recognition (CVPR), 2020 - Oral. [Github page](https://github.com/gistvision/e2sri)
```

Comment:
We only write "Oral" to link to a video of the presentation.

The git repo is empty... are you sure you want to link it at this point in time or when it is populated?

Reply:
Got it thanks, then lets just update as:
"IEEE Conf. Computer Vision and Pattern Recognition (CVPR), 2020."
without the repo and oral at the moment. I will put a new request once the video and repo are completed.

Reply:
So, are you committing the change?

Reply:
Yes please.
But please update the change in L658 to "IEEE Conf. Computer Vision and Pattern Recognition (CVPR), 2020." that is without putting the Github link and without stating it is oral.